docs/product/performance/assets.mdx

```diff
@@ -1,20 +1,20 @@
 ---
-title: Resources
+title: Assets
 sidebar_order: 50
-description: "Learn more about browser resource performance monitoring, which allows you to debug the performance of loading JavaScript and CSS on your frontend."
+description: "Learn more about browser asset performance monitoring, which allows you to debug the performance of loading JavaScript and CSS on your frontend."
 ---
 
-If you have performance monitoring enabled for your frontend, you can see how your browser resources are performing in Sentry.
+If you have performance monitoring enabled for your frontend, you can see how your browser assets are performing in Sentry.
```

Comment:
Thanks for doing this so fast @DominikB2014 🎉 

Mind adding a banner that mentions we've renamed this page, something like:

> Note: This product feature was originally named "Resources", "Assets" has the same functionality with a new _exciting_ name.

Maybe someone else can come up with a more humorous blurb that gets the point across.

Reply:
Yeah, will do!